src/Auth0.OidcClient.WPF/WebViewBrowser.cs

```diff
@@ -72,9 +80,19 @@ public Task<BrowserResult> InvokeAsync(BrowserOptions options, CancellationToken
             };
 
             window.Show();
+
+#if NET6_0
+            await webView.EnsureCoreWebView2Async();
+
+            // Delete existing Cookies so previous logins won't remembered
+            webView.CoreWebView2.CookieManager.DeleteAllCookies();
```

Comment:
I know this is in the official sample, and I have used it before. I was still wondering if we should do this.

Have you been able to use this without this one line? What is the behavior when doing so?

Reply:
If you don't call EnsureCoreWebView2Async(), then CoreWebView2 will be null.

Reply:
I mean the deleting of the cookies.

Reply:
Ah sorry for the misunderstanding.
I added it because it came like that in the example you quoted. I'm going to check it by deleting it and doing logout and then login again without closing my WPF application, to see if the cookies are kept...

Reply:
It is not necessary because the window is created each time and no cookies are kept, so I have removed it.